test/no-focused-test.spec.js

```diff
@@ -0,0 +1,69 @@
+const { RuleTester } = require('eslint');
+const rule = require('../lib/rules/no-focused-test');
+
+RuleTester.setDefaultConfig({
+  parserOptions: {
+    ecmaVersion: 2018,
+  },
+});
+
+const wrapInTest = (name, input) => `test('verify noFocusedTest rule with ${name}', async () => { ${input} })`;
```

Comment:
Since we are verifying the test cases themselves, it probably doesn't need the `wrapInTest` stuff.